public/base-locales/de.json

```diff
@@ -559,5 +559,9 @@
   "seen_nearby_cell": "in dieser S2 Zelle gesehen",
   "seen_lure_encounter": "getroffen am Lockmodul",
   "seen_lure_wild": "Spawn am Lockmodul",
-  "seen_wild": "Spawn in der Wildnis"
+  "seen_wild": "Spawn in der Wildnis",
+  "historic_rarity": "Historische Rarität",
+  "poi": "Sehenswürdigkeiten",
```

Comment:
It's related to Wayfarer... It's not even point of interest... It's called "Wayspots"

Reply:
And also in english

https://niantic.helpshift.com/hc/en/21-wayfarer/section/167-how-to-review-wayspots/